fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalJDBCScanOperator.java

```diff
@@ -0,0 +1,61 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer.operator.logical;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.JDBCTable;
+import com.starrocks.catalog.Table;
+import com.starrocks.sql.optimizer.operator.OperatorType;
+import com.starrocks.sql.optimizer.operator.OperatorVisitor;
+import com.starrocks.sql.optimizer.operator.Projection;
+import com.starrocks.sql.optimizer.operator.scalar.ColumnRefOperator;
+import com.starrocks.sql.optimizer.operator.scalar.ScalarOperator;
+
+import java.util.Map;
+
+public class LogicalJDBCScanOperator extends LogicalScanOperator {
```

Comment:
Better to add equal and hashcode methods，otherwise you're going to bury a bug

Reply:
> Better to add equal and hashcode methods，otherwise you're going to bury a bug

@Youngwb No new fields are introduced here, I think inheriting the implementation of the parent class is enough.